shared/css/sidebar.css

```diff
@@ -259,42 +259,57 @@ html[dir="rtl"] .text-chat-entry.received {
   margin-right: 10px;
 }
 
-.text-chat-entry > p {
-  position: relative;
-  z-index: 10;
+.text-chat-entry p {
```

Comment:
> All that said, I will concede that inconsistent style sucks, though to some degree, that ship has already sailed in our CSS. I'll revert these changes if necessary if you still feel strongly about it (do you?), but I just felt the need to grumble about us being held back by an obsolete tradition. Thank you for tolerating my crankiness. :-)

Given the extra pages referenced, I can agree that we don't need to use child selectors everywhere. I'd say that normally we discuss these sort of changes to our procedures/policy before doing them (and there was at least some agreement a while ago to generally switch to child selectors), so please bring this up at the retrospective at least, so that we can all be on the same page.

I would say that I think we do need to continue to bring in the consistency around what CSS applies where - layout within a component to be on that component, layout of a component within a parent component to be applied via the parent. I think that's the biggest thing that'll help us.

Re-reading some of the css here makes it look like we won't really be able to apply that to the text chat if speech-bubble is separated out, but it looks like we probably need to re-architect the rendering this component anyway, so maybe that doesn't matter at the moment.